libraries/liveIntentId/idSystem.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * This module adds LiveIntentId to the User ID module.
+ * The {@link module:modules/userId} module is required.
+ * @module modules/idSystem
+ * @requires module:modules/userId
+ */
+import { triggerPixel, logError } from '../../src/utils.js';
+import { ajaxBuilder } from '../../src/ajax.js';
+import { gdprDataHandler, uspDataHandler, gppDataHandler } from '../../src/adapterManager.js';
+import { submodule } from '../../src/hook.js';
+import { LiveConnect } from 'live-connect-js'; // eslint-disable-line prebid/validate-imports
+import { getStorageManager } from '../../src/storageManager.js';
+import { MODULE_TYPE_UID } from '../../src/activities/modules.js';
+import { DEFAULT_AJAX_TIMEOUT, MODULE_NAME, composeIdObject, eids, DEFAULT_DELAY, GVLID, PRIMARY_IDS, parseRequestedAttributes } from './shared.js'
```

Comment:
Hi. You are importing the `DEFAULT_DELAY` const but it does not exist.

Reply:
https://github.com/prebid/Prebid.js/pull/12334